programs/ssl/ssl_mail_client.c

```diff
@@ -57,8 +57,10 @@
 #include <unistd.h>
 #else
 #include <io.h>
-#define read _read
-#define write _write
+#define read(fd, buf, len)                                      \
+    _read( fd, (void *)( buf ), (unsigned int)( len ) )
```

Comment:
Theoretically, we could lose data here. parameter `len` is `size_t`, which is 64 bit in x64. if for some reason, len would be MAX_UINT, we will downcast to unsigned int and lose information. This casting only removes the warning(symptom), not the data loss.

Reply:
at least comment this, so people using this sample code would be aware of this risk